src/baguetteBox.js

```diff
@@ -156,9 +156,9 @@
 
     function bindImageClickListeners(selector, userOptions) {
         // For each gallery bind a click event to every image inside it
-        var gallery = document.querySelectorAll(selector);
-        galleries.push(gallery);
-        [].forEach.call(gallery, function(galleryElement) {
+        var nodeList = document.querySelectorAll(selector);
```

Comment:
I wanted to keep in gallery var the DOM element and the eventHandler. Take a look in f67a3a08c9829196098f8b8ed6a656dbdaa78b84.